src/types/shopping-events.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+
+import type { UserData  } from './user';
+
+export type EventType = 'pageView' | 'purchase' | 'setUser' | 'cancelCart' | 'customEvent';
+export type PageType = 'HOME_PAGE' | 'CATEGORY' | 'SEARCH_RESULTS' | 'DEALS' | 'CART' | 'CHECKOUT' | 'ORDER_CONFIRMATION';
+
+export type Category = {|
+    id? : string,
+    name? : string
+|};
+
+export type PageView = {|
+    id? : string,
+    page_type? : PageType,
```

Comment:
`page_type` is required

Reply:
I'll update the doc, to make it optional. I'd rather keep it optional at 1st release